rulesbook/codegen/guides/quality/README.md

```diff
@@ -0,0 +1,629 @@
+| ✨ Success | 0 | Positive reinforcement (no penalty) |
+
+### Score Thresholds
+
+| Score Range | Status | Action |
+|-------------|--------|--------|
+| 95-100 | ✨ Excellent | Auto-approve, document success patterns |
+| 80-94 | ✅ Good | Approve with minor notes |
+| 60-79 | ⚠️ Fair | Approve but recommend fixes |
+| 40-59 | ❌ Poor | Block until critical issues fixed |
+| 0-39 | 🚨 Critical | Immediate block, requires rework |
+
+### Example Scoring
+
+```
+Scenario: Connector with quality issues
+
+Critical Issues: 1 (Wrong UCS trait used)    = -20 points
+Warning Issues: 2 (Code duplication)         = -10 points
+Suggestions: 5 (Documentation improvements)  = -5 points
+
+Quality Score = 100 - 20 - 10 - 5 = 65
+
+Result: ⚠️ FAIR - Pass with warnings
+```
+
+---
+
+## 📚 Key Resources
+
+### Primary Resources
+
+| File | Purpose | When to Use |
+|------|---------|-------------|
+| [feedback.md](../feedback.md) | Master feedback database with review template | Reference before/during implementation |
+| [quality_review_template.md](quality_review_template.md) | Standalone review template | For Quality Guardian during reviews |
+| [CONTRIBUTING_FEEDBACK.md](CONTRIBUTING_FEEDBACK.md) | Guide for adding feedback | When adding new patterns/issues |
+
+### Reference Documentation
+
+| Document | Purpose |
+|----------|---------|
+| [patterns/](../patterns/) | Flow-specific implementation patterns |
+| [connector_integration_guide.md](../connector_integration_guide.md) | Complete integration guide |
+| [types/types.md](../types/types.md) | UCS type system reference |
+| [learnings/learnings.md](../learnings/learnings.md) | Implementation lessons learned |
+
+---
+
+## 🚀 Quick Start Guide
+
+### For Quality Guardian Subagent
+
+**Before Each Review:**
+```markdown
+1. Read guides/feedback.md completely
+2. Extract patterns relevant to current phase/flow
+3. Prepare quality checklist from applicable feedback
+4. Load quality_review_template.md
+```
+
+**During Review:**
+```markdown
+1. Analyze code against UCS patterns
+2. Check Rust best practices
+3. Validate connector patterns
+4. Count issues by severity
+5. Calculate quality score
+6. Fill out quality_review_template.md
+```
+
+**After Review:**
+```markdown
+1. Generate review report
+2. Make pass/warn/block decision
+3. Update feedback.md with new patterns
+4. Increment frequency for observed patterns
+5. Document success patterns
+```
+
+### For Developers
+
+**Before Implementation:**
+```markdown
+1. Review Section 1: Critical Patterns in feedback.md
+2. Read flow-specific patterns for your flow
+3. Understand common anti-patterns to avoid
+4. Reference success patterns for inspiration
+```
+
+**After Quality Review:**
+```markdown
+1. Read quality review report carefully
+2. Fix all CRITICAL issues immediately
+3. Address WARNING issues before next phase
+4. Consider SUGGESTIONS for improvement
+5. Learn from feedback for next flow
+```
+
+**When Adding Feedback:**
+```markdown
```

Comment:
This is already in a markdown file. Why do we need a code snippet for markdown of list? why not just render the list directly?